vignettes/custom_appearance.Rmd

```diff
@@ -90,3 +94,289 @@ lyt3 <- basic_table() %>%
+
+Within the layout initializer:
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("STRATA1") %>%
+  analyze("AGE") %>%
+  append_topleft("New top_left material here")
+
+build_table(lyt, DM)
+```
+
+Specify label position using the `split_rows` function. Notice the position of `STRATA1` and `SEX`.
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("STRATA1", label_pos = "topleft") %>%
+  split_rows_by("SEX", label_pos = "topleft") %>%
+  analyze("AGE")
+
+build_table(lyt, DM)
+```
+
+Post-processing using the `top_left()` function:
+
+```{r}
+lyt <- basic_table() %>%
+  split_cols_by("ARM") %>%
+  split_rows_by("SEX") %>%
+  analyze(c("AGE", "STRATA1"), function(x) {
+    if (is.numeric(x)) {
+      in_rows(
+        "mean" = rcell(mean(x)),
+        "sd" = rcell(sd(x)),
+        .formats = c("xx.x"), .aligns = "left"
+      )
+    } else if (is.factor(x)) {
+      rcell(length(unique(x)), align = "right")
+    } else {
+      stop("Unsupported type")
+    }
+  }, show_labels = "visible", na_str = "NE") %>%
+  build_table(ex_adsl)
+
+# Adding top-left material
+top_left(lyt) <- "New top-left material here"
+
+lyt
+```
+
+### Table Inset
+
+Table title, table body, referential footnotes and and main footers can be inset from the left alignment of the titles and provenance footer materials. This can be modified within the layout initializer `basic_table()` using the `inset` argument or during post-processing with `table_inset()`.
+
+Using the layout initializer:
+
+```{r}
+lyt <- basic_table(inset = 5) %>%
+  analyze("AGE")
+
+build_table(lyt, DM)
+```
+
+Using the post-processing function:
+
+Without inset - 
+```{r}
+lyt <- basic_table() %>%
+  analyze("AGE")
+
+tbl <- build_table(lyt, DM)
+tbl
+```
+
+With an inset of 5 characters - 
+
+```{r}
+table_inset(tbl) <- 5
+tbl
+```
+
+
+Below is an example with a table produced for clinical data. Compare the inset of the table and main footer between the two tables.
+
+Without inset -
+```{r}
+analysisfun <- function(x, ...) {
+    in_rows(row1 = 5,
+            row2 = c(1, 2),
+            .row_footnotes = list(row1 = "row 1 rfn"),
+            .cell_footnotes = list(row2 = "row 2 cfn"))
+    }
+
+lyt <- basic_table(title = "Title says Whaaaat", subtitles = "Oh, ok.",
+                   main_footer = "ha HA! Footer!", prov_footer = "provenaaaaance") %>%
+    split_cols_by("ARM") %>%
+    analyze("AGE", afun = analysisfun)
+
+result <-  build_table(lyt, ex_adsl)
+result
+```
+
+With inset - 
+Notice, the inset does not apply to any title materials (main title, subtitles, page titles), provenance footer materials and any horizontal dividers. Inset settings is applied to top-left materials, referential footnotes and main footer materials.
```

Comment:
inset does apply to horizontal dividers, see the output in my comment above and please modify this text